data/net/minecraft/world/level/gameevent/GameEvent.mapping

```diff
@@ -0,0 +1,115 @@
+		COMMENT This event is broadcasted when an entity falls far enough to take fall damage.
+	FIELD LIGHTNING_STRIKE Lnet/minecraft/world/level/gameevent/GameEvent;
+		COMMENT This event is broadcasted when lightning strikes a block.
+	FIELD MOB_INTERACT Lnet/minecraft/world/level/gameevent/GameEvent;
+		COMMENT This event is broadcasted when a player interacts with a mob. For example when the player heals a tamed wolf or feeds an animal.
+	FIELD PISTON_CONTRACT Lnet/minecraft/world/level/gameevent/GameEvent;
+		COMMENT This event is broadcasted when a piston head is retracted.
+	FIELD PISTON_EXTEND Lnet/minecraft/world/level/gameevent/GameEvent;
+		COMMENT This event is broadcasted when a piston head is extended.
+	FIELD PRIME_FUSE Lnet/minecraft/world/level/gameevent/GameEvent;
+		COMMENT This event is broadcasted when an entity such as a creeper or TNT begins exploding.
+	FIELD PROJECTILE_LAND Lnet/minecraft/world/level/gameevent/GameEvent;
+		COMMENT This event is broadcasted when a projectile hits something.
+	FIELD PROJECTILE_SHOOT Lnet/minecraft/world/level/gameevent/GameEvent;
+		COMMENT This event is broadcasted when a projectile is fired.
+	FIELD RAVAGER_ROAR Lnet/minecraft/world/level/gameevent/GameEvent;
+		COMMENT This event is broadcasted when the ravager uses it's roar ability.
+	FIELD RING_BELL Lnet/minecraft/world/level/gameevent/GameEvent;
+		COMMENT This event is broadcasted when the bell block is rung.
+	FIELD SHEAR Lnet/minecraft/world/level/gameevent/GameEvent;
+		COMMENT This event is broadcasted when a shear is used. This includes disarming tripwires, harvesting honeycombs, carving pumpkins, etc.
+	FIELD SHULKER_CLOSE Lnet/minecraft/world/level/gameevent/GameEvent;
+		COMMENT This event is broadcasted when a shulker closes it's shell.
+	FIELD SHULKER_OPEN Lnet/minecraft/world/level/gameevent/GameEvent;
+		COMMENT This event is broadcasted when a shulker opens it's shell.
+	FIELD SPLASH Lnet/minecraft/world/level/gameevent/GameEvent;
+		COMMENT This event is broadcasted wen an entity splashes in the water. This includes boats paddling or hitting bubble columns.
+	FIELD STEP Lnet/minecraft/world/level/gameevent/GameEvent;
+		COMMENT This event is broadcasted when an entity moves on the ground. This includes entities such as minecarts.
+	FIELD SWIM Lnet/minecraft/world/level/gameevent/GameEvent;
+		COMMENT This event is broadcasted as an entity swims around in water.
+	FIELD WOLF_SHAKING Lnet/minecraft/world/level/gameevent/GameEvent;
+		COMMENT This event is broadcasted when a wolf shakes off water after getting wet.
+	FIELD name Ljava/lang/String;
+		COMMENT The name of the event. This is primarily used for debugging game events. @see net.minecraft.client.renderer.debug.GameEventListenerRenderer#render
+	FIELD notificationRadius I
+		COMMENT The radius around an event source to broadcast this event. Any listeners within this radius will be notified when the event happens.
```

Comment:
I generally try to document everything even if it's a bit redundant.

Reply:
These javadocs are completely fine, and I think it's good to have since they provide a bit of context outside of just the name. Which makes it clearer what exactly some of these events "covers"